Shared/Flows/Onboarding/OnboardingCoordinator.swift

```diff
@@ -43,9 +43,14 @@ extension OnboardingCoordinator: LaunchActivityHandler {
 
 extension OnboardingCoordinator: OnboardingViewControllerDelegate {
     
-    nonisolated func onboardingView(_ controller: OnboardingViewController,
-                                    didVerify user: User) {
+    nonisolated func onboardingViewController(_ controller: OnboardingViewController,
+                                              didOnboard user: User) {
         Task {
+            await controller.showLoading()
+            try await ActivateUser().makeRequest(andUpdate: [],
+                                                 viewsToIgnore: [controller.view])
+            await controller.hideLoading()
+
```

Comment:
Once onboarding is complete, we'll activate the user. If the user is on the waitlist, this function won't get called.